net/net-snmp/files/snmpd.conf

```diff
@@ -116,4 +116,5 @@ config engineid
 #	option trapsess	'-v 3 -e 0x80001f88808c18d3f7b0000 -u trapuser -a MD5 -A administrator -l authPriv -x DES -X rootpasswd udp:127.0.0.1:162'
 
 config snmpd general
+	option enable '1'
```

Comment:
A quick grep shows that enabled is mostly used instead of enable. I am sucker for consistency, so please use enabled here as well.